tests/React.Tests/Mvc/HtmlHelperExtensionsTests.cs

```diff
@@ -250,7 +255,8 @@ public void RenderFunctionsCalledNonLazily()
 				new { },
 				null,
 				false,
-				true
+				true,
+				false
```

Comment:
Let's use named arguments for all these booleans to make it clearer (eg. `skipLazyInit: false`)

Reply:
Regrettably, msbuild does not allow that in this case.

> An expression tree may not contain a named argument specification